### PR TITLE
Organizer action view

### LINKED ISSUE #1098: Interface for handling flagged calls
## Description
An organizer should be able to find calls that have been flagged for organizer action (OAN, _organizer action needed_), and take care of them. This should be done using Views, by adding a new view column type, and a link to create such a view from the call assignment page.

## Relevant Job Stories
None (because we weren't using Job Stories in as much of a structured way in R36 when this was first researched). Will potentially write some, and add them here later.

## Prerequisites
The column type must first be implemented on the backend, before proceeding with this.

## Requirements
- [x] There should be a new column type added for displaying OAN messages
  - [x] The cell displays number of unsolved actions, and an exclamation point on an orange background
- [x] Hovering the cell should display a popper
  - [x] Display still open messages (where action is needed) first
  - [x] Display number of solved issues (if any)
  - [x] Clicking a button should open a pane
- [x] Pane should list notes on calls, sorted by status (unsolved/solved) and date
  - [x] Each note has the name and (relative) date of the call
  - [x] Each note has the text message from the caller
  - [x] Each note has a button to toggle whether action has been taken or not
- [x] The status card on a call assignment summary page should include a button to create a view
  - [x] The new view should use Smart Search to find people with OAN
  - [x] The new view should include a column of the new type
  - [x] The new view should be sorted such that unsolved flaggings are on top

## Possible implementations
Update whether action has been taken by patching `/orgs/ID/calls/ID`.

### View column
image

### Call assignment status card
image

## Open questions
None

### PR DESCRIPTION
**Pull request by niklasva82**
## Description
This PR adds View features (column and Smart Search filter) to display a list of all people who are being blocked from calling due to "organizer action needed", and a pane for removing the flagging.

## Screenshots
image

## Changes
* Adds support for new `organizer_action` view column type
* Adds an RPC to create a special view for this use case, or return an existing one if one exists
* Adds Smart Search filter for `call_blocked`

## Notes to reviewer
Requires changes to backend that are not yet deployed on the dev server.

## Related issues
Resolves #1098

### CODE REVIEW

**Comment by niklasva82** [comment]
Not sure what to do about this.
bild

I'm just doing the same thing as createNew/server.ts in order to get messages server-side in the getOrganizerAction rpc.

**Comment by richardolsson** [comment]
@niklasva82 The problem you had when trying to internationalize your RPC is that the same RPC module imported both server-side and client-side code, and was used in the browser (which is throwing an error because the module is trying to import server-only code, `fs`).

This is solved by separating the client-side code from the server-side code in the RPC as I've done here.

File: src/features/callAssignments/models/CallAssignmentModel.ts [hunk 1]
```diff
@@ -207,6 +207,18 @@ export default class CallAssignmentModel extends ModelBase {
...
+    this._repo.updateCall(this._orgId, callId, {
+      organizer_action_taken: false,
+    });
+  }
+
+  setOrganizerActionTaken(callId: number) {
+    this._repo.updateCall(this._orgId, callId, {
+      organizer_action_taken: true,
+    });
+  }
```
Comment (richardolsson) [inline src/features/callAssignments/models/CallAssignmentModel.ts:220]: These aren't used anywhere, the ones in `CallModel` are.

**Review by richardolsson — CHANGES_REQUESTED** [review]
I have now looked through all the code more thoroughly (after just getting it to work when I first took over the PR).

The comments below are the things I found that I will be changing. I'm just leaving them as comments to share the feedback.

**Review by richardolsson — APPROVED** [review]
I've fixed all the issues I found in my previous review so I'm going to go ahead and merge this now. 😊

**Pull request merged**